libs/shared/src/containers/TransactionList/TransactionLine/index.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styles from './styles.module.less';
+import { Grid } from 'semantic-ui-react';
+
+type VisualProps={
+  key: number,
+  locale: string,
+
+  yearToDisplay: number,
+  monthTodisplay: string,
+  dayToDisplay: number,
```

Comment:
Why did you in date this way, rather than just passing in a `DateTime`?

Reply:
Wanted to have a very stupid component just for the display